.eslintrc.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
It was not my intention to change the formatting, etc. VSCode's default formatter is throwing an error, so I switched to Prettier. But that should be fine as we're using it in other repos as well, right?